lib/ASTGen/Sources/ASTGen/Literals.swift

```diff
@@ -55,14 +55,14 @@ extension ASTGenVisitor {
     let stringLiteralKind = node.stringLiteralKind ?? .singleLine
     let delimiterLength = node.delimiterLength
     let startLoc = self.generateSourceLoc(node)
-    let afterQuoteLoc: BridgedSourceLoc = {
+    let afterQuoteLoc: SourceLoc = {
       var l = startLoc
       if let pound = node.openingPounds {
-        l = l.advanced(by: pound.trimmedLength.utf8Length)
-        l = l.advanced(by: pound.trailingTriviaLength.utf8Length)
-        l = l.advanced(by: node.openingQuote.leadingTriviaLength.utf8Length)
+        l = l.advanced(by: Int32(pound.trimmedLength.utf8Length))
```

Comment:
`CInt` is more robust. But I think it's reasonable to change the C++ side to accept `size_t` instead of `int`

Reply:
There are a couple of places where we advance by a negative offset. I did try switching to `intptr_t`, but that required some additional static casts that I reserved for a subsequent PR, in calls that rely on `unsigned` -> `int` wrapping, as in `loc.getAdvancedLoc(-presumedLoc.getColumn() + 1)`.

Reply:
You could also use `ssize_t` which imports as `Int`

Reply:
I tried `intptr_t` because 

```
typedef intptr_t SwiftInt;
typedef uintptr_t SwiftUInt;
``` 

I’m tempted to move these to `Basic/SwiftBridging.h` and use `SwiftInt` for consistency.